src/components/crown/crownButtons/crownTaskDropdown.vue

```diff
@@ -42,7 +43,7 @@ export default {
         this.$emit('toggle-dropdown-state', false);
         return;
       }
-      this.$emit('replace-node', data);
+      this.$emit('confirm-replace', data);
```

Comment:
I think the event name should still be 'replace-node'. The fact that we're confirming the replacement is an implementation detail, the action is still replacing the node.

Reply:
I can't seem to get it to work that way. It seems to emit up the `replace-node` regardless of my interception and destroys the node before the confirmReplace method ever fires.

Reply:
I could put the confirmReplace on Modeler.vue

Reply:
OK, that's fine. My main problem is more the name of the event: `confirm-replace` seems to me to be too closely tied to the implementation. Can you think of something that's more generic. describing the intent rather than the specifics?